svg/elements/feGaussianBlur.json

```diff
@@ -21,7 +21,7 @@
               "version_added": true
             },
             "ie": {
-              "version_added": null
+              "version_added": "10"
```

Comment:
Testing https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feGaussianBlur#Example in IE 9 and 10 as you did in https://github.com/mdn/browser-compat-data/issues/7006 confirms this.

One wrinkle is that the element and its interface seems to be supported in IE 9 per http://staging-dot-mdn-bcd-collector.appspot.com/tests/api/SVGFEGaussianBlurElement, but that's not a useful level of support, so I agree with setting "10".